### ISSUE TITLE
npm: Failed at the hiredis@0.1.17 install script 'node-gyp rebuild'

### ISSUE THREAD

**Issue by kristianmandrup** [redis/hiredis-node#94]
Please resolve this issue. Thanks :)

```
gyp ERR! node -v v2.3.4
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok 
npm ERR! Darwin 14.4.0
npm ERR! argv "/usr/local/bin/iojs" "/Users/kristianmandrup/npm/bin/npm" "install"
npm ERR! node v2.3.4
npm ERR! npm  v2.13.0
npm ERR! code ELIFECYCLE

npm ERR! hiredis@0.1.17 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hiredis@0.1.17 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the hiredis package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls hiredis
npm ERR! There is likely additional logging output above.
```

**Comment by kristianmandrup** [comment]
Tried to force dependency to 0.4.0

```sh
npm ERR! node v2.3.4
npm ERR! npm  v2.13.0
npm ERR! code ELIFECYCLE

npm ERR! hiredis@0.4.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hiredis@0.4.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the hiredis package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
```

**Comment by badboy** [comment]
Can you please give the output of `npm install --loglevel=verbose hiredis`?

**Comment by msifd** [comment]
3.x.x python is not supported by node-gyp.

**Comment by Dindaleon** [comment]
Has this been solved? I cannot install hiredis either.

**Comment by badboy** [comment]
1. Use the latest hiredis-node release
2. Make sure you have Python 2
3. Make sure you're node-gyp is not broken.

Then it should work. If not, please report back with a verbose log output.

**Comment by kalmecak** [comment]
I fix it using hiredis 0.4.1
